documentation/assemblies/assembly-metrics-rules.adoc

```diff
@@ -2,6 +2,8 @@
 //
 // assembly-monitoring.adoc
 
+:parent-context: {context}
```

Comment:
the templates have changed and this might make things better:

at the top:

`// Retains the context of the parent assembly if this assembly is nested within another assembly.
// For more information about nesting assemblies, see: https://redhat-documentation.github.io/modular-docs/#nesting-assemblies
// See also the complementary step on the last line of this file.
ifdef::context[:parent-context: {context}]`

at the bottom

`// Restore the context to what it was before this assembly.
ifdef::parent-context[:context: {parent-context}]
ifndef::parent-context[:!context:]`